src/lib/components/video-player.svelte

```
@@ -0,0 +1,97 @@
+<script lang="ts">
+import Hls from "hls.js";
+import { onMount, onDestroy } from "svelte";
```

ESLint

[error] 'onMount' is defined but never used.

(@typescript-eslint/no-unused-vars)